pkgs/development/python-modules/linien-common/default.nix

```diff
@@ -38,6 +38,13 @@ buildPythonPackage rec {
     appdirs
   ];
 
+  # Temporarily fix rpyc version requirement before upstream switch to latest
+  # See: https://github.com/linien-org/pyrp3/pull/10#pullrequestreview-1585887668
```

Comment:
Hmm I'm a bit confused, how come `linien-common` builds, while our rpyc version is `>5.0`?

Reply:
Oh now I see that on branch `master`, it doesn't build, due to the version requirement not met... I wonder how come `hydra-check python310Packages.linien-common` shows differently...

In anycase, I'm thinking of perhaps introducing a `rpyc4` package instead as the issue is with the compatibility with the server Python, which currently uses a a Python version which supports only rpyc4... That's what upstream tried to explain in https://github.com/linien-org/pyrp3/pull/10#discussion_r1302816237

Reply:
Error also occurs using `nix build nixpkgs#pkgsCross.armv7l-hf-multiplatform.python3Packages.linien-common`.

Reply:
> In anycase, I'm thinking of perhaps introducing a `rpyc4` package instead as the issue is with the compatibility with the server Python, which currently uses a a Python version which supports only rpyc4... That's what upstream tried to explain in [linien-org/pyrp3#10 (comment)](https://github.com/linien-org/pyrp3/pull/10#discussion_r1302816237)

Understood. Will disregard this PR now.